ci_framework/roles/devscripts/molecule/default/converge.yml

```diff
@@ -19,7 +19,7 @@
   hosts: all
 
   vars:
-    ansible_user_dir: "{{ lookup('env', 'HOME') }}"
+    ansible_user_dir: "{{ ansible_env.HOME }}"
```

Comment:
Is it being moved from controller over to target?

[docs](https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-access-shell-environment-variables)

Reply:
Initial design was to run this on the controller however it has changed for `reproducer` hence replacing all `lookup` to enable supporting of remote execution also.

Reply:
ack, thx